util/type_traits.h

```diff
@@ -53,13 +53,13 @@ using has_atomic_support =
 /**
  * @brief Checks whether T is a floating-point sycl type
  */
-template <typename T>
-using is_cl_float_type =
-    std::bool_constant<std::is_floating_point<T>::value ||
-                       std::is_same<cl::sycl::half, T>::value ||
-                       std::is_same<cl::sycl::cl_float, T>::value ||
-                       std::is_same<cl::sycl::cl_double, T>::value ||
-                       std::is_same<cl::sycl::cl_half, T>::value>;
+template <typename T> struct is_cl_float_type {
```

Comment:
I don't know it from the diffs, but is `is_cl_float_type` still being used?

Reply:
Great catch, thank you. Yes, `is_cl_float_type` is still being used in SYCL-CTS code base along with deprecated `cl_*` types. After removing this typed we will still need this type trait for the `sycl::half` type. Probably we will rename this type trait from `is_cl_float_type` to `is_sycl_float_type`.